docs/mkdocs.yml

```diff
@@ -76,20 +76,20 @@ nav:
       - Getting Started with Lance: quickstart/index.md
       - Versioning: quickstart/versioning.md
       - Vector Search: quickstart/vector-search.md
-    - Format Specification: format/index.md
```

Comment:
the leveling was a bit odd right now, seems like user guide, format spec and community guide should be nested in getting started. fixed along the way.